shared-bindings/qrio/QRInfo.c

```diff
@@ -63,3 +63,39 @@ const mp_obj_namedtuple_type_t qrio_qrinfo_type_obj = {
         MP_QSTR_data_type,
     },
 };
+
+//| class QRPosition:
+//|     """Information about a non-decoded QR code"""
+//|
```

Comment:
Please include the fields here so that it is stubbed out fully.

Reply:
Oh, sorry, I forgot to do this.